scripts/bootstrap.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -e # Exit as soon as any line in the bash script fails
+
+# Colors for error/success messages
+GREEN="\033[0;32m"
+YELLOW="\033[0;33m"
+NO_COLOR="\033[0m" # No color
+
+cd $(dirname "$0")/..
+
+# Remove all yarn locks
+find . -type f -name 'yarn.lock' -not -path "*/node_modules/*" -exec sh -c 'x="{}"; mv "$x" "${x}_"' \;
+# Remove packages's modules
+find . -type d -name "node_modules" -prune -exec rm -rf {} \;
+
+# Install dependenices
+echo -e "${YELLOW}Installing dependencies...${NO_COLOR}"
+yarn install --silent
+
+# Build all packages
+echo -e "${YELLOW}Building packages...${NO_COLOR}"
+# Build mongoose first in order to be dependencey of @adminjs/import-export
+yarn workspace @adminjs/mongoose build
+yarn workspaces --no-progress run build
+
+# Build AdminJS
+echo -e "${YELLOW}Building AdminJS...${NO_COLOR}"
+yarn workspace adminjs bundle:globals
```

Comment:
I'm not sure if `build` and `bundle:globals` generates only production bundle or both dev and prod?

Reply:
I am not sure if I understood well :) Here `build` and `bundle:globals` is to initialize (bootstrap) and is run without `NODE_ENV=production` so generate `global-bundle.development.js` bundle (if you eg want to just run an example app). If you would work on admin js in dev mode then you should run `bundle` (is then in dev mode with reloads)